skill_settings:
  reaching:
    init:  # initial joint configurations of the robot
      A: [-0.4089345703125, -0.5016806640625, -1.6099755859375, 1.174388671875, 1.0947353515625, 1.78128125, -1.509703125]
      B: [0.21878125, -0.4029912109375, -1.5717705078125, 1.981884765625, 1.1924970703125, 1.4775771484375, -1.6183740234375]
      C: [-0.7226513671875, -0.3468486328125, -1.943255859375, 0.4815615234375, 1.775826171875, 2.0602998046875, -0.981400390625]
    env: # array of object poses vectorizes as [x,y,z,x,y,z,w]
      A: [[0.663151784935, -0.0728068145558, 0.339698737234, 0.624733496663, 0.780478570194, 0.0159083805715, 0.0175551429265]]
      B: [[0.588961357829, 0.171217158472, 0.338809974027, 0.662607625885, 0.747621786525, -0.0340778089284, 0.0291804964059]]
      C: [[0.62770756706, 0.419988270742, 0.335795757158, 0.71182206267, 0.702099107925, -0.00630604886881, 0.0180673046022]]

  stacking:
    init:
      A: [-0.4089345703125, -0.5016806640625, -1.6099755859375, 1.174388671875, 1.0947353515625, 1.78128125, -1.509703125]
      B: [0.21878125, -0.4029912109375, -1.5717705078125, 1.981884765625, 1.1924970703125, 1.4775771484375, -1.6183740234375]
      C: [-0.7226513671875, -0.3468486328125, -1.943255859375, 0.4815615234375, 1.775826171875, 2.0602998046875, -0.981400390625]
      D: [-0.772646484375, -0.6264443359375, -1.6202177734375, 0.5026396484375, 1.16376953125, 2.1437177734375, -1.2865625]
    env:
      A: [[0.569977608025, 0.151495446323, 0.337571196285, 0.999129805722, -0.000152108666142, -0.0376034491974, 0.0180440790542]]
      B: [[0.732561217433, 0.201135069746, 0.334039275857, 0.99889215227, -0.0370070858297, -0.0284792557112, -0.00582028577845]]
      C: [[0.636550641343, 0.263951542667, 0.335549993572, 0.998877786168, -0.000758696236108, -0.0429446425489, 0.0199587163179]]

  closing:
    init: 
      A: [-0.4089345703125, -0.5016806640625, -1.6099755859375, 1.174388671875, 1.0947353515625, 1.78128125, -1.509703125]
      B: [0.21878125, -0.4029912109375, -1.5717705078125, 1.981884765625, 1.1924970703125, 1.4775771484375, -1.6183740234375]
      C: [-0.7226513671875, -0.3468486328125, -1.943255859375, 0.4815615234375, 1.775826171875, 2.0602998046875, -0.981400390625]
    env:
      A: [[0.75086393155, 0.0983334701844, 0.221951339181, 0.939054268912, -0.339207311966, -0.0543507413416,  -0.0127073389654]]
      B: [[0.760541860532, 0.231048776137, 0.21443412917, 0.863007659015, 0.504581206608, -0.01830014314, -0.0167538407658]]
      C: [[0.732440491405, 0.0950148220513, 0.21311402779, 0.952306614261, -0.304053823334, -0.0255759912794, -0.00304197639619]]

  pressing:
    init:
      A: [-0.4089345703125, -0.5016806640625, -1.6099755859375, 1.174388671875, 1.0947353515625, 1.78128125, -1.509703125]
      B: [0.21878125, -0.4029912109375, -1.5717705078125, 1.981884765625, 1.1924970703125, 1.4775771484375, -1.6183740234375]
      C: [-0.612025390625, -0.7637890625, -0.22923046875, 0.2971669921875, 0.17831640625, 2.035369140625, -0.6385625]#[-0.7226513671875, -0.3468486328125, -1.943255859375, 0.4815615234375, 1.775826171875, 2.0602998046875, -0.981400390625]
    env:
      A: [[0.726212485704, -0.45337885777, 0.367190146878, 0.931002157073, -0.364460226582, -0.0160626749651, -0.0120713395837]]
      B: [[0.679193184254, -0.264332047375, 0.364756331947, 0.990677643956, -0.135172131869, -0.00613521702183, 0.0157689455328]]
      C: [[0.74516219077, -0.346413616193, 0.365174736251, 0.782210198086, -0.622902871223, -0.00242143060302, -0.0115479741331]]

  cartpushing:
    init:
      A: [-0.7208955078125, -0.38083984375, -0.23441015625, 0.49685546875, 0.210025390625, 1.389447265625, -0.7158603515625]
      B: [-0.88645703125, -0.4723193359375, -0.196818359375, 0.371384765625, 0.07548046875, 1.7404189453125, -0.8156748046875]
      C: [-1.2027978515625, -0.638806640625, -0.1553466796875, 0.2753759765625, 0.0839892578125, 1.929822265625, -1.146892578125]
      D: [-1.10251953125, -0.8555888671875, -0.1906279296875, 1.4974990234375, 0.1368466796875, 0.902666015625, -1.13160546875]
    env:
      A: [[0,0,0,0,0,0,1]]
      B: [[0,0,0,0,0,0,1]]
      C: [[0,0,0,0,0,0,1]]


